extensions/resource-deployment/src/ui/deployClusterWizard/pages/azureSettingsPage.ts

```diff
@@ -26,48 +26,94 @@ export class AzureSettingsPage extends WizardPageBase<DeployClusterWizard> {
 		const azureSection: SectionInfo = {
 			title: '',
 			labelPosition: LabelPosition.Left,
-			fields: [
-				{
+			spaceBetweenFields: '5px',
+			rows: [{
+				fields: [{
 					type: FieldType.Text,
 					label: localize('deployCluster.SubscriptionField', "Subscription id"),
 					required: false,
 					variableName: SubscriptionId_VariableName,
 					placeHolder: localize('deployCluster.SubscriptionPlaceholder', "Use my default Azure subscription"),
 					description: localize('deployCluster.SubscriptionDescription', "The default subscription will be used if you leave this field blank.")
 				}, {
+					type: FieldType.ReadonlyText,
+					label: '',
+					labelWidth: '0px',
+					defaultValue: localize('deployCluster.SubscriptionHelpText', "{0}"),
```

Comment:
>localize('deployCluster.SubscriptionHelpText', "{0}"), [](start = 19, length = 54)

You're missing params here so this is just going to display `{0}` which probably isn't what you want. (not sure what the value is supposed to be but if there's no other text then there's no point in passing this through localize anyways)

Reply:
it is actually the placeholder for the links.